### PR TITLE
Javalab: default to test Javabuilder stack on localhost

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Quick config change to make working with Javalab easier locally. With this, Javalab will point the `javabuilder-test` stack on localhost by default, so that developers don't need to spin up localhost Javabuilder or manually override the URL in cdo.rb. Developers can set the `use_localhost_javabuilder` flag in locals.yml to instead point to localhost Javabuilder, or set `local_javabuilder_stack_name` to point to a deployed dev instance of Javabuilder.

I'll also bump up the limits on the javabuilder-test configuration separately so we don't run into execution limits too easily (PR [here](https://github.com/code-dot-org/javabuilder/pull/359)).

## Testing story

Tested locally, using all three configurations (default using test stack, using localhost javabuilder, point to dev stack).

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: lib/cdo.rb [hunk 1]
```diff
@@ -148,9 +153,14 @@ def javabuilder_url(path = '', scheme = '')
...
-        # 'https://<your-javabuilder-domain>-http.dev-code.org/seedsources/sources.json'
-        'http://localhost:8080/javabuilderfiles/seedsources'
+        # On localhost, we default to using the "test" Javabuilder stack. To point
+        # to your Javabuilder WebSocket server running on localhost, set
+        # 'use_localhost_javabuilder: true' in your locals.yml. To point to a
+        # deployed development instance of Javabuilder, set
+        # 'local_javabuilder_stack_name: "your stack name"' in your locals.yml.
+        return 'http://localhost:8080/javabuilderfiles/seedsources' if CDO.use_localhost_javabuilder
+        stack_name = CDO.local_javabuilder_stack_name || 'javabuilder-test'
+        "https://#{stack_name}-http.code.org/seedsources/sources.json"
```
Comment (molly-moen) [inline lib/cdo.rb:163]: we may have to split this out if we end up doing adhocs on the dev environment, but I think we are sticking with prod for now.

**Review by molly-moen — APPROVED** [review]
Nice improvement!

File: lib/cdo.rb (hunk 1 above)
Comment (sanchitmalhotra126) [reply lib/cdo.rb:163]: Ah yep good call. We can update this logic once we switch over

**Comment by bencodeorg** [comment]
From the peanut gallery, love this change 🙌

**Comment by fisher-alice** [comment]
> From the peanut gallery, love this change 🙌

Such a helpful change - thanks!

**Pull request merged**